### ISSUE TITLE
Cannot helper issue

### ISSUE THREAD

**Issue by Exelord** [minutebase/ember-can#67]
Hi,
I noticed some unexpected errors according to `cannot` helper with looking for a wrong `__container__`. I fix it by defining my own helper:

### Before
<img width="542" alt="screen shot 2017-09-19 at 17 00 44" src="https://user-images.githubusercontent.com/1469375/30599248-2e28ed9c-9d5c-11e7-9ed4-c2c7b353ad5e.png">

### My Solution

```js
// helpers/cannot.js

import CanHelper from 'ember-can/helpers/can';

export default CanHelper.extend({
  compute() {
    return !this._super(...arguments);
  }
});

```

What do you think about it?

**Comment by mupkoo** [comment]
This solution is really nice and solves my problem as well

**Comment by sukima** [comment]
Is this project dead? This was reported 5 months ago and no word. This issue illustrates a **fundamental** failure of the helper itself. I would consider this a _severe_ bug.

With the proposed solution being so simple and demonstrating the unnecessary use of complex `getOwner` logic can we get a fix?! At this point our project is littered with `(not (can "..."))` everywhere! It looks bad.

CC @blimmer

**Comment by Exelord** [comment]
Yep it looks dead. If you would have same time we can fork it, fix update and revive. I doubt the author will cover it :/

**Comment by blimmer** [comment]
@sukima I'm sorry but I don't have time to work on this project anymore. I would recommend @Exelord 's suggestion of forking to revive. An alternative would be to reach out to @rlivsey (who has admin on the repo) to see if he would give either of you access to directly work with this codebase.

The last time I had talked with @rlivsey he didn't have time to work on this project, either.

**Comment by sukima** [comment]
@blimmer 

I am more than happy to help. I can also maintain either by forking or becoming an admin. The only question is how to handle releases through NPM.

**Comment by blimmer** [comment]
Unfortunately I don't have the powers to give you either of those things 😞 

@rlivsey would need to help out here.

**Comment by Exelord** [comment]
That would be cool. I can also help with that :)

**Comment by sukima** [comment]
@Exelord is this you? https://keybase.io/exelord

-----BEGIN PGP MESSAGE-----
Version: Keybase OpenPGP v2.0.76
Comment: https://keybase.io/crypto

wcFMAy+CpCoBJi9vAQ/7BxoWAT1WiSXpPfgAd8JSw/36mtXxD/jB1fa+mXHoTCmU
vx/YY3wiq5ZgeBkWwz12XU6B+qwrySHXhTjJ2LbYu6+lorMqwVdjL6pRknSIHS6O
7I4oTw9vt+piZemrQa1EKZmBKc2HpxB69nsTyXqj2wzWyNo+ru7tJrCxNs6uifdC
8C01WTUodBbuF1QGeLVV6cUzWULuL/vjh4GpvdeD4mLiaLCxcJHsVHhpr+f1yE+i
0YZjJ6dmwzuDp6V2zOc2c2PpZv2Lroi9zeY3yNPZLPaPQP/2wqOerquiwW4AwDMF
17oH93Aq2QioqVOSasoAFWc8Pgg2hHkPNjAqcUn5R1CWsR6kZteFhMGeGrGRDTLG
sC/pTMi9uWqCcTr81FIRmqW2+YslKdtn1ptZT6k63HA9X1Nco7F02H/SFRbYm2SO
I3J/9VLJv694Qe8Z/fQdJuF3WLvYSBdB8yDGCCIbYBHdRk9QrxCeHR7PAjhbJiVF
l+FMRL8Ffx+cgVjpeE1kuxIRybi1c9Qpnj6iEnzcGyLPZRlatm23K3Paqp+VYf3Q
JpUWNcQ3SD4IfxzIbTQJVo7bHbUkuRZgBBXNVPFxdpDxothbESLd0z0D+h3lAW+7
gDynNSOy32i01g+h/PIwCmZ3kjYqEwUSD+G7qOfk4fd/a9S4W/lhNtpmBnMjNRDS
gwGkouGWEwFQNwb786vt13c3OPtweNv2EvdIYpJGp7qhmp1+CxvYtTwevyB97MXR
m+9mXtuS6jyQupr+dhDO1DvpsUviWPQwheGKBBoOhGDv5ZyTeTBqVvxt7y+8NZWe
te6BGqcRAE64AkdVMBVJ1j1i/IpKV1TUwsHuigmjJ84DY5dW
=vPR8
-----END PGP MESSAGE-----

**Comment by Exelord** [comment]
https://gist.github.com/Exelord/33974317856e390fa3107a02e50d5f0f

**Comment by rlivsey** [comment]
I’m alive, sorry for the neglect, it’s been an … interesting … year. Feeling less burned out now and going to get back on top of things ASAP. 

At Ember conf right now, if anyone else is here feel free to grab me. Happy to add more people with commit bits here & on npm the get things back on track.

**Comment by Exelord** [comment]
I'm watching it 😅. Do you think it's still possible to discuss? 😉

**Comment by bravo-kernel** [comment]
Good to hear you are coming back to life mr. @rlivsey. I found out first hand that one of the best things to prevent 🔥 and ⚰️ is spreading responsibilities. Please take the time to grant permissions, this repo deserves it.

**Comment by sukima** [comment]
@rlivsey I'm at EmberConf too!!!

**Comment by sukima** [comment]
@rlivsey Sent you a keybase.io chat. ([keybase/GitHub proof](https://gist.github.com/sukima/77b9e1e69dcc17c3eea3))

**Comment by Exelord** [comment]
Fixed in #74
